### PR TITLE
fix pep8 for cloudscale_server

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
fix pep8 

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
cloudscale_server

##### ANSIBLE VERSION

```
2.4
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by Akasurde — APPROVED** [review]
shipit

**Comment by resmo** [comment]
@Akasurde thanks for the review ;)

File: lib/ansible/modules/cloud/cloudscale/cloudscale_server.py [hunk 1]
```diff
@@ -376,44 +355,39 @@ def create_server(self):
...
+            image=dict(),
+            volume_size_gb=dict(type='int', default=10),
+            bulk_volume_size_gb=dict(type='int'),
+            ssh_keys=dict(type='list'),
+            use_public_network=dict(type='bool', default=True),
+            use_private_network=dict(type='bool', default=False),
+            use_ipv6=dict(type='bool', default=True),
+            anti_affinity_with=dict(),
+            user_data=dict(),
+            api_token=dict(no_log=True),
```
Comment (gaudenz) [inline lib/ansible/modules/cloud/cloudscale/cloudscale_server.py:390]: While agree to most of the changes this one has very dubious value IMO. It certainly does not improve readability.

Comment (resmo) [reply lib/ansible/modules/cloud/cloudscale/cloudscale_server.py:390]: you mean "no_log=True"? This was already used before and prevents secrets being logged.

Comment (gaudenz) [reply lib/ansible/modules/cloud/cloudscale/cloudscale_server.py:390]: No I mean this whole chunk of the diff. Unfortunately github is not really clever in displaying this. My opinion is that aligning the = makes the code much more readable than strict adherence to PEP8.

**Pull request merged**